tests/app/main/test_suppliers.py

```diff
@@ -1714,55 +1716,71 @@ def test_should_be_an_error_if_missing_or_invalid_duns_number(self, duns_number)
         )
 
     def test_should_be_an_error_if_duns_number_in_use(self):
-        self.data_api_client.find_suppliers.return_value = {
-            "suppliers": [
-                "one supplier", "two suppliers"
-            ]
-        }
-        res = self.client.post(
-            "/suppliers/create/duns-number",
-            data={
-                'duns_number': "123456789"
-            }
-        )
+        with self.app.app_context():
+            with mock.patch(self.direct_plus_api_method, return_value={'primaryName': 'COMPANY LTD'}):
```

Comment:
Not going to assert the calls made to this mock?

Reply:
Same for `data_api_client`

Reply:
Yeah, will write another test for that methinks.

Reply:
@risicle 
https://github.com/alphagov/digitalmarketplace-supplier-frontend/pull/1150/commits/3310e7ecf1077928ade6354f76a793b324d8eb35